### PR TITLE
Fix #1 ; Fix #6 ; Feature 1,2,3 added and extra features

### LINKED ISSUE #6: Draw Game
shows player 1 wins for DRAW match.

### PR DESCRIPTION
**Pull request by iosdev474**

### CODE REVIEW

**Comment by iosdev474** [comment]
closing due to adding all 3 features in other pull request

**Comment by iosdev474** [comment]
please ignore commit messages :p and fix #1 , fix #6 and 
Added Feature 1,2,3
Added Features:
	Player should have different names
	Player should have different symbols
	Player's name can not be blank
	Player's symbol can not be blank
	Player can choose Alphabets,numbers etc as their symbol

File: TicTacToe/src/tictactoe/GameWindow.java [hunk 1]
```diff
@@ -26,18 +26,43 @@
     private JLabel playingLabels[][] = new JLabel[3][3];
     private final int PLAYINGAREAX = 110, PLAYINGAREAY = 100, CELLSIZE = 40, SPACEBETWEENCELLS = 10;
     private int currentPlayer = 0;
-    
+    int moved[][]=new int[3][3];
```
Comment (nikilagar) [inline TicTacToe/src/tictactoe/GameWindow.java]: Why not boolean.

File: TicTacToe/src/tictactoe/GameWindow.java [hunk 2]
```diff
@@ -46,10 +71,28 @@ private void endGame(int currentPlayer){
...
+        boolean draw=true;
+        for(int i=0;i<3;i++)
+        {
+            for(int j=0;j<3;j++)
+                if(moved[i][j]==0)
+                {
+                    draw=false;
+                }
+        }
+        if(draw==true)
```
Comment (nikilagar) [inline TicTacToe/src/tictactoe/GameWindow.java]: No need to compare with true. If(draw) would suffice!!!

File: TicTacToe/src/tictactoe/GameWindow.java [hunk 3]
```diff
@@ -150,32 +213,201 @@ public GameWindow() {
...
+                        .addComponent(p2Label)
+                        .addComponent(p2Field, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE)
+                        .addComponent(changeButton)))
+                .addContainerGap())
         );
 
         pack();
     }// </editor-fold>//GEN-END:initComponents
 
+    private void nameTextfieldActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_nameTextfieldActionPerformed
```
Comment (nikilagar) [inline TicTacToe/src/tictactoe/GameWindow.java]: Unnecessary function.

File: TicTacToe/src/tictactoe/GameWindow.java [hunk 4]
```diff
@@ -150,32 +213,201 @@ public GameWindow() {
...
+        // TODO add your handling code here:
+    }//GEN-LAST:event_nameTextfieldActionPerformed
+
+    private void nextButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_nextButtonActionPerformed
+    if(!nameTextfield.getText().equals(""))
```
Comment (nikilagar) [inline TicTacToe/src/tictactoe/GameWindow.java]: Indent the code.

File: TicTacToe/src/tictactoe/GameWindow.java [hunk 5]
```diff
@@ -150,32 +213,201 @@ public GameWindow() {
...
+        p1Label.setText(name1);
+    }
+    else
+    {
+        JOptionPane.showMessageDialog(null,"Please choose a valid name.");
+    }
+    }//GEN-LAST:event_nextButtonActionPerformed
+
+    private void startButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_startButtonActionPerformed
+    if(!nameTextfield.getText().equals(""))
```
Comment (nikilagar) [inline TicTacToe/src/tictactoe/GameWindow.java]: Indent the code.

**Comment by urdarinda** [comment]
@nikilagar @iosdev474 My eyes hurt by seeing the title and the commit messages. Please fix :-1:

**Review by nikilagar — CHANGES_REQUESTED** [review]

File: TicTacToe/src/tictactoe/GameWindow.java (hunk 1 above)
Comment (iosdev474) [reply TicTacToe/src/tictactoe/GameWindow.java]: ok

File: TicTacToe/src/tictactoe/GameWindow.java (hunk 2 above)
Comment (iosdev474) [reply TicTacToe/src/tictactoe/GameWindow.java]: ok

File: TicTacToe/src/tictactoe/GameWindow.java (hunk 3 above)
Comment (iosdev474) [reply TicTacToe/src/tictactoe/GameWindow.java]: removed

File: TicTacToe/src/tictactoe/GameWindow.java (hunk 4 above)
Comment (iosdev474) [reply TicTacToe/src/tictactoe/GameWindow.java]: done indenting

File: TicTacToe/src/tictactoe/GameWindow.java (hunk 5 above)
Comment (iosdev474) [reply TicTacToe/src/tictactoe/GameWindow.java]: done indenting

**Comment by iosdev474** [comment]
adding another feature so closing request :-)

**Pull request closed without merging**